mcp/kubernetes_locals.tf

```diff
@@ -42,13 +42,13 @@ locals {
     if ! contains(split("/", kube_file), "terraform")
```

Comment:
This was an old idea when we thought we were looking for files per micro service but really we should be explicit. Line 38 should just look for the file in the root directory. See https://github.com/mesoform/terraform-infrastructure-modules/blob/master/mcp/locals.tf#L16

Reply:
Does this apply only to k8s_secret.yml, or to all yml configuration description files?

Reply:
All files